CHANGES_NEXT_RELEASE

```diff
@@ -3,3 +3,5 @@ Fix:  Changed the description text for parse errors in v2 json requests (Issue #
 Fix:  Remove leftover whitespaces in error messages (Issue #1199) 
 Fix:  Avoid updating several matching entities associated to the same ID, returning TooManyResults error in that case (Issue #1198)
 Fix:  Permitting attribute values to be empty in API v2 (Issue #1188)
+Add:  Using options=count when getting entites (#1041)
```

Comment:
In addition, options=count may lead to think that options= has to be used literarly that way. Maybe is more precise:

Using 'count' option when getting entities.